examples/misc_object_update_system.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>three.js misc - object update system</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <style>
+        body {
+            color: #000;
+            font-family:Monospace;
+            font-size:13px;
+            text-align:center;
+
+            background-color: #fff;
+            margin: 0px;
+            overflow: hidden;
+        }
+
+        #info {
+            position: absolute;
+            top: 0px; width: 100%;
+            padding: 5px;
+        }
+
+        a {
+            color: #0af;
+        }
+
+        #stats { position: absolute; top:0; left: 0 }
+        #stats #fps { background: transparent !important }
+        #stats #fps #fpsText { color: #777 !important }
+        #stats #fps #fpsGraph { display: none }
+    </style>
+</head>
+
+<body>
+
+<div id="container"></div>
+
+<div id="info">
+    <a href="http://threejs.org" target="_blank">three.js</a> misc - object update system</div>
+
+<script src="../build/three.min.js"></script>
+
+<script src="js/Detector.js"></script>
+<script src="js/libs/stats.min.js"></script>
+
+<script>
+
+    var SCREEN_WIDTH = window.innerWidth;
+    var SCREEN_HEIGHT = window.innerHeight;
+    var FLOOR = -250;
+
+    var container,stats;
+
+    var camera, scene;
+    var renderer;
+
+
+    var mouseX = 0, mouseY = 0;
+
+    var windowHalfX = window.innerWidth / 2;
+    var windowHalfY = window.innerHeight / 2;
+
+    var clock = new THREE.Clock();
+
+    container = document.getElementById( 'container' );
+
+    camera = new THREE.PerspectiveCamera( 30, SCREEN_WIDTH / SCREEN_HEIGHT, 1, 10000 );
+    camera.position.z = 150;
+
+    scene = new THREE.Scene();
+
+    scene.add( camera );
+
+
+    // RENDERER
+
+    renderer = new THREE.WebGLRenderer( { antialias: true } );
+    renderer.setClearColor( '#000');
```

Comment:
`renderer.setClearColor( 0x000000 );` But black is the default, so the line is not necessary.